jmxtrans-output/jmxtrans-output-core/pom.xml

```diff
@@ -41,6 +41,10 @@
 			<artifactId>commons-lang</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>commons-logging</groupId>
```

Comment:
Why did you add this dependency to commons-logging? It seems that there is no code using it directly...